static/src/javascripts/projects/commercial/modules/comscore.spec.ts

```diff
@@ -76,14 +92,14 @@ jest.mock('../../common/modules/commercial/commercial-features', () => ({
 describe('setupComscore', () => {
 	it('should do nothing if the comscore is disabled in commercial features', () => {
 		commercialFeatures.comscore = false;
-		setupComscore();
+		void setupComscore();
```

Comment:
I think you’d need to either `await` this, or return it and chain the expect check inside the `then`